### DISCUSSION TITLE
Deck Multi-ranger and Loco Positioning System?

### DISCUSSION THREAD

**Discussion by lauribeiro** [bitcraze/discussions#475] [Q&A] [answered]
Hello!
I'm a student who has just started working with Crazyflie 2.1.
I want to work with the Multi-ranger and the Loco Positioning system, but some small doubts have arisen along the way.
The LPS is already working perfectly. And I have searched the forum and found that they can be used together... Here some a few questions:
1. What is the best way to expand these decks?
2. Can I put any at the bottom? Is there a problem using the multi-ranger without the flow deck?
3. Has anyone tried this? Is there any kind of background material? 

Thank you in advance! :)

Sorry if these questions have already been answer.

**Comment by krichardsson** [comment, marked as answer]
Hi!

It should be fine to use the Loco and Multiranger decks at the same time. You can put the decks both at the top and bottom in any combination, but the top sensor of the multiranger will obviously not give you any useful data if mounted under the CF. The Loco deck works under the CF as well, but reception might be a problem on the floor depending on the material of the floor. If take off is a problem you can put the CF on a small cardbox for instance to get it some distance up and improve the reception.
The Mutiranger works fine without the Flow deck, they are often used together since it makes a nice combination, but there is no problem using them separately or in combination with other decks
I don't know if Loco + multiranger has been used before, it sounds like an interesting thing to try out though :-)

**Reply by lauribeiro** [reply]
Thank you so much for the answers and the tips. 
I will keep trying to develop the project. When (and if) it has evolved, I'll come back to share.
Thank you! :)
